app/controllers/comments_controller.rb

```diff
@@ -10,6 +10,7 @@ def create
 
     if comment.save
       redirect_to user_post_path(current_user.id, post)
+      flash[:success] = 'Comment was successfully added.'
     else
       render :new
```

Comment:
- [ ] We are expected to add flash messages for both successful and not successful paths. Please add another flash message in the else block just in case the creation of the comment was not successful. This is necessary for full interactivity and enhancement of the user experience.

- [ ] Please handle the same issue as above in the posts controller.

- [ ] Lastly, this review expected to see the `likes controller` and how you've handled the creation of a like in the controller. There is no `likes` controller in the controller's directory currently. Please add this controller, it will be necessary for the creation of a like.

Reply:
Done